cfgov/v1/models/home_page.py

```diff
@@ -33,11 +33,37 @@ class Meta:
         }
 
 
+class HighlightCardValue(blocks.StructValue):
+    @property
+    def link_text(self):
+        return 'Read more'
```

Comment:
How will we handle translations with this?

Reply:
Good call. Happily [we already translate](https://github.com/cfpb/consumerfinance.gov/blob/169420b38960b5377d546220dc9de9be34e23f30/cfgov/locale/es/LC_MESSAGES/django.po#L397-L399) this phrase, so we can use that.